front/app/components/admin/GranularPermissions/PermissionCardButton/index.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Box,
+  Icon,
+  Text,
+  Title,
+  IconNames,
+  colors,
+} from '@citizenlab/cl2-component-library';
+import React, { useState } from 'react';
+import { MessageDescriptor } from 'react-intl';
+import { FormattedMessage } from 'utils/cl-intl';
+import { lighten } from 'polished';
+
+type PermissionCardButtonProps = {
+  onClick: () => void;
+  selected: boolean;
+  iconName?: IconNames;
+  title: MessageDescriptor;
+  subtitle: MessageDescriptor;
+};
+export const PermissionCardButton = ({
+  onClick,
+  selected,
+  iconName,
+  title,
+  subtitle,
+}: PermissionCardButtonProps) => {
+  const [isHover, setIsHover] = useState(false);
+
+  // Temporary colour until new tealLight variations added to component library
+  const backgroundColor = lighten(0.1, colors.tealLight);
```

Comment:
I haven't received the new colours yet to add to the component library. Maybe I can just add a TODO here for now?